frontend/testing/playwright/tests/ui-editor/ui-editor-data-model-binding-and-gitea.spec.ts

```diff
@@ -10,17 +10,23 @@ import { Header } from '../../components/Header';
 import { DataModelPage } from '../../pages/DataModelPage';
 import { GiteaPage } from '../../pages/GiteaPage';
 
+const getAppTestName = (app: string) => `bindings-${app}`;
+
+const WAIT_ONE_SECOND = 2000;
```

Comment:
Is this for testing purposes within the pipeline? 😊 It's frustrating to encounter waits in our tests... If this resolves the issue, it's worth dedicating some time to understand why they're necessary.

Reply:
I just refactored it into a variable in this PR :) it was already implemented to make sure we are able to actually do the actions before the test move on. I explained the issue on Slack earlier today😄